experiment_code/hackrl/experiment.py

```diff
@@ -46,6 +46,7 @@
 
 class TtyrecEnvPool:
     def __init__(self, flags, dataset_name, dataset_scores, **dataset_kwargs):
+        self.flags = flags
```

Comment:
Maybe its better just to pass this argument omitted_dlvls?
`self.omitted_dlvls = omitted_dlvls`